src/react/components/MnemonicGenerationFlow/3-VerifyMnemonic.tsx

```diff
@@ -20,6 +20,7 @@ type VerifyMnemonicProps = {
  * @returns react element to render
  */
 const VerifyMnemonic: FC<VerifyMnemonicProps> = (props): ReactElement => {
+  let inputFields:Array<React.MutableRefObject<HTMLInputElement | undefined>> = [];
```

Comment:
I know I helped you here, but I think we should change this line to:

`let inputFields:Array<React.MutableRefObject<HTMLInputElement>> = [];`

now that will cause an issue at line 102 because we are potentially adding an undefined value but we'll cast it to be defined all the time as it should be.

The new line 102 would be:
`inputFields.push(inputRef as React.MutableRefObject<HTMLInputElement>);`